hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/handle/HandleWorkflowTest.java

```diff
@@ -335,7 +353,8 @@ void testContructorWithInvalidArguments() {
                         serviceLookup,
                         configProvider,
                         instantSource,
-                        null))
+                        null,
+                        exchangeRateManager))
```

Comment:
We can also add a test with _null_ exchangeRateManager

Reply:
Added